apis/lightclient/proof.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  operationId: getProof
+  summary: Get multiproof of `paths` for `state_id`
+  description: |
+    Returns a multiproof of `paths` at the requested `stateId`.
+    The requested `stateId` may not be available. Regular nodes only keep recent states in memory.
+  tags:
+    - Lightclient
+  parameters:
+    - name: state_id
+      in: path
+      required: true
+      $ref: '../../beacon-node-oapi.yaml#/components/parameters/StateId'
+    - name: paths
+      in: query
+      required: true
+      description: Array of JSON serialized array of paths
+      schema:
+        type: array
+        items:
+          type: string
+          description: |
+          example: "'[\"balances\", 0]'"
+           
+  responses:
+    "200":
+      description: Success
+      content:
+        application/octet-stream:
```

Comment:
The JSON representation of this format is widely inefficient. Since JSON is the default I'll argue into only offering `application/octet-stream` to force consumers to use the efficient path